### ISSUE TITLE
Load accounts settings doesn't work

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#273]
When trying to restore a template using the "Load Account Settings" while including the tabs from the license tab, it doesn't overwrite existing settings for the account:

![image](https://github.com/RealRaven2000/SmartTemplates/assets/6314674/b835f240-315e-49e1-b386-5245399bfd7a)

This is probably a side effect from rewriting code to use IOUtils.

In error console we can see the following message:

```
Uncaught (in promise) SyntaxError: JSON.parse: unexpected non-whitespace character after JSON data at line 1 column 4 of the JSON data
    readData chrome://smarttemplate4/content/settings.js:1347
    readSuccess chrome://smarttemplate4/content/settings.js:1433
settings.js:1347:27

```

**Comment by RealRaven2000** [comment]
here is quick fix - replaced `IOUtils.read` with `IOUtils.readJson` and removed the JSON parser:

[smartTemplate-fx-4.3.3pre5.zip](https://github.com/RealRaven2000/SmartTemplates/files/13249051/smartTemplate-fx-4.3.3pre5.zip)

----

to install the test version, download the zip file and then drag the file into Thunderbird Add-ons Manager (do not extract contents).

**Comment by RealRaven2000** [comment]
Fixed in v 4.3.3 released Nov 8th.
